include/robot_interfaces/opencv_driver.hpp

```diff
@@ -0,0 +1,73 @@
+#pragma once
+
+#include <ctime>
+#include <iostream>
+
+#include <real_time_tools/process_manager.hpp>
+#include <real_time_tools/thread.hpp>
+#include <real_time_tools/threadsafe/threadsafe_timeseries.hpp>
+#include <real_time_tools/timer.hpp>
+#include "opencv2/opencv.hpp"
+
+namespace robot_interfaces
+{
+/**
+ * @brief Driver for interacting with any camera using OpenCV.
+ */
+
+template <typename OpenCVObservation>
+class OpenCVDriver
+{
+public:
+    cv::VideoCapture video_capture;
+    real_time_tools::Timer timer;
+
+    OpenCVDriver()
+    {
+        cv::VideoCapture cap(0);
+        video_capture = cap;
```

Comment:
Better initialize `video_capture` directly with the desired argument:
```suggestion
    OpenCVDriver():
        video_capture(0)
    {
```

To be a bit more flexible, it might be nice to pass the argument to the constructor (can have `0` as default value). This would be relevant when multiple cameras are connected.